packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ScatterPlotViewer/ScatterPlotViewer.stories.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import zooTheme from '@zooniverse/grommet-theme'
+import { Box, Grommet } from 'grommet'
+import { withKnobs, boolean, text, number } from '@storybook/addon-knobs';
+import ScatterPlotViewer from './ScatterPlotViewer'
+import ZoomInButton from '../../../ImageToolbar/components/ZoomInButton/ZoomInButton'
+import ZoomOutButton from '../../../ImageToolbar/components/ZoomOutButton/ZoomOutButton'
+import ResetButton from '../../../ImageToolbar/components/ResetButton/ResetButton'
+import mockData from '../LightCurveViewer/mockData'
+import readme from '../LightCurveViewer/README.md'
+import backgrounds from '../../../../../../../.storybook/lib/backgrounds'
+
+const config = {
+  // notes: {
+  //   markdown: readme
+  // }
+}
+
+const transformMatrixMock = {
+  scaleX: 1,
+  scaleY: 1,
+  skewX: 0,
+  skewY: 0,
+  translateX: 0,
+  translateY: 0
+}
+
+let zoomCallback
+
+function onZoom (type) {
+  zoomCallback(type)
+}
+
+function setZoomCallback (callback) {
+  zoomCallback = callback
+}
+
+const stories = storiesOf('ScatterPlotViewer', module)
+
+stories.addDecorator(withKnobs)
+
+stories
+  .add('light theme', () => {
```

Comment:
Small point, but you can declare a default viewport here with `addParameters`. to save some repeated declarations later. Also, `storiesOf()` methods chain so this could be written as
```js
storiesOf('ScatterPlotViewer', module)
  .addDecorator(withKnobs)
  .addParameters({ viewport: { defaultViewport: 'responsive' }})
  .add(…)
```